### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by dpolozhyi**

### CODE REVIEW

File: JinnSports.DAL/Migrations/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            //  You can use the DbSet<T>.AddOrUpdate() helper extension method 
+            //  to avoid creating duplicate seed data. E.g.
+            //
+            //    context.People.AddOrUpdate(
+            //      p => p.FullName,
+            //      new Person { FullName = "Andrew Peters" },
+            //      new Person { FullName = "Brice Lambson" },
+            //      new Person { FullName = "Rowan Miller" }
+            //    );
+            //
```
Comment (NikitaMagda) [inline JinnSports.DAL/Migrations/Configuration.cs]: Are you sure that you need this comment?

File: JinnSports.Entities/Entities/Team.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@ namespace JinnSports.Entities.Entities
     public class Team
     {
         public int Id { get; set; }
+        
         public virtual SportType SportType { get; set; }
         public string Name { get; set; }
```
Comment (NikitaMagda) [inline JinnSports.Entities/Entities/Team.cs]: Blank line between properties

**Comment by dpolozhyi** [comment]
Added code-based migration to Migrations folder in DAL. Also edit JsonParser to work with new entities

File: JinnSports.Parser.App/JsonParsers/JsonParser.cs [hunk 3]
```diff
@@ -96,14 +95,14 @@ public List<Result> GetResultsList(JsonResult result)
             List<SportType> sportList = new List<SportType>();
             foreach (var e in result.Events)
             {
-                Team team1 = new Team() { Results = new List<Result>() };
-                Team team2 = new Team() { Results = new List<Result>() };
+                Team team1 = new Team();
+                Team team2 = new Team();
                 if (this.GetTeamsFromEvent(e, team1, team2))
```
Comment (NikitaMagda) [inline JinnSports.Parser.App/JsonParsers/JsonParser.cs]: As I remember you can declare out parameters like
if (this.GetTeamsFroomEvent(e, Team team1, Team team2))

**Pull request merged**